content/docs/command-reference/add.md

```diff
@@ -73,19 +68,21 @@ large files. DVC also supports other link types for use on file systems without
 
 ### Tracking directories
 
-A `dvc add` target can be an individual file or a directory. In the latter case,
-a `.dvc` file is created for the top of the directory (with default name
-`<dir_name>.dvc`).
-
-Every file in the hierarchy is added to the cache (unless the `--no-commit`
-option is used), but DVC does not produce individual `.dvc` files for each file
-in the directory tree. Instead, the single `.dvc` file references a special JSON
-file in the cache (with `.dir` extension), that in turn points to the added
-files.
-
-Note that DVC commands that use tracked files support granular targeting of
-files, even when the directory is added as a whole. Examples: `dvc push`,
-`dvc pull`, `dvc get`, `dvc import`, etc.
+A `dvc add` target can be either a file or a directory. In the latter case, a
+`.dvc` file is created for the top of the hierarchy (with default name
+`<dir_name>.dvc`). Every file inside is added to the cache (unless the
```

Comment:
add `(exception .dvcignore)`

Reply:
.dvcignore is mentioned earlier, probably no need to complicate this section about directories? Plus I think you can still add files in .dvcignore as long as it's done explicitly. Or it was Git that allows that, and I opened an issue about it in the core repo? Can't remember right now.

Reply:
Let's discuss in https://github.com/iterative/dvc.org/pull/1384#pullrequestreview-461181992 🙂 (resolving this one).